docs/storage/index.md

```diff
@@ -50,7 +50,6 @@ The use of storage is billed according to the [billing policy](../runjobs/lumi_e
 
 === "Project persistent"
 
-    Persistent storage intended to share data amongst the members of a project.
```

Comment:
I do no see anything misleading in this sentence.

Reply:
I would also change 

> project expires

to

*project ends*

Reply:
I changed the 'expired' -> 'ends'

Why I removed this sentence "Persistent storage intended to share data amongst the members of a project." is because I don't like it that it mentions "intended to share data", which might make some people think that it's the place where their I/O data for the project should be placed

Reply:
Only that they might ask us to extend the size of the storage area to fit their data there :) 
Also the word "persistent" gives a signal that e.g. /scratch is not persistent. And leads to think that it's possible that it will be automatically cleaned at some point (for which there is a risk that it will, but for now this doesn't seem probable)

Reply:
Could this be somehow rephrased, if you want to keep that sentence?

Reply:
Than, I would change the "tab" name from "project persistent" to "project space"; add additional sentence like: "Transient data is meant to be processed with scratch space which allows higher quotas."

Reply:
Changed the tab name as suggested and added the sentence. Also changed the reference to 'persistent' elsewhere on the page + on the 'Jupyter for courses' page where it was mentioned (+ corrected the link to point to the correct tab)